src/components/Comments/Comment.js

```diff
@@ -291,14 +315,6 @@ const Comment = ({
                 </ButtonHightlight>
               </TouchableOpacity>
             )}
-            <TouchableOpacity activeOpacity={1} onPress={onPress} testID="memoComment">
-              <ButtonHightlight
-                onLongPress={handleOnLongPress}
-                style={[styles.btnReply, styles.iconContainer]}
-                onPress={onPress}>
-                <MemoCommentReply height={normalize(14)} />
-              </ButtonHightlight>
-            </TouchableOpacity>
           </>
         )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The structure of the component has been changed. Ensure that these changes do not break any existing functionality or layout.
```diff
+                  </ButtonHightlight>
+                </TouchableOpacity>
+              )}
+            </>
+          )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->